src/commands/synthetics/interfaces.ts

```diff
@@ -91,10 +92,16 @@ export interface MultiStepsTestResult extends TestResult {
 
 export type Result = BrowserTestResult | ApiTestResult | MultiStepsTestResult
 
+interface Enrichment {
+  batch_id: string
+  config_override: ConfigOverride & {executionRule: ExecutionRule}
+}
+
 export interface PollResult {
   check?: Test
   check_id?: string
   dc_id: number
+  enrichment?: Partial<Enrichment>
```

Comment:
What are the cases where `enrichment` would not be defined? Could that cause incorrect `executionRule` evaluation?

Reply:
As far as I can tell, it should always be defined. Though to minimise as much as possible the changes on this PR, I set it as optional. I can make it a required property, but with a bit of "noise" for the review (i.e. adding `enrichment: {},` in some places) 👀

Reply:
Good to me to leave as optional then 👍